README.md

```diff
@@ -129,8 +129,8 @@ In this example `move-result` is constantized, as well as `return` because there
 .end method
 ```
 
-Dead code includes:
+Common dead code includes:
 
-* unreferenced assignments - assigning something and never using it
-* method calls with no side-effects - `Ljava/lang/Integer;->intValue()I` has no side-effects
+* unreferenced variables through assignments - assigning something and never using it
```

Comment:
Not just variables, any opcode that does assignment could be removed if it's unused and there are no side effects.